mkiocccentry.c

```diff
@@ -7003,3 +6992,65 @@ remind_user(char const *work_dir, char const *entry_dir, char const *tar, char c
     }
     return;
 }
+
+static char *
+cmdprintf(int err, const char *func, const char *format, ...)
+{
+    va_list va;
+    size_t size = 0;
+    const char *next, *p, *f;
+    const char *esc = "\t\n\r !\"#$&()*;<=>?[\\]^`{|}~";
+    char *d, *cmd, c;
+    int nquot;
+
+    va_start(va, format);
+    f = format;
+    while ((c = *f++))
+	if (c == '%') {
+	    p = next = va_arg(va, const char*);
+	    nquot = 0;
+	    while ((c = *p++))
+		nquot = c == '\'' ? 
+		    size += nquot > 1 ? 3 : nquot + 1, 0 :
+		    nquot + !!strchr(esc, c);
+	    size += (nquot > 1 ? 2 : nquot) + (p - next) - 2;
+	}
+    va_end(va);
+    size += f - format;
+
+    cmd = malloc(size);
+    if (cmd == NULL) {
+	errp(err, func, "malloc from the cmdprintf of %ld bytes failed", size);
+	/*NOTREACHED*/
+    }
```

Comment:
The indentation of the closing brace looks strange here.

Reply:
This is because github uses a tab with 4 spaces and the mkiocccentry source code uses a tab with 8 spaces.